clients/java/signalr/src/main/java/com/microsoft/signalr/DefaultHttpClient.java

```diff
@@ -24,10 +27,10 @@
 
 final class DefaultHttpClient extends HttpClient {
     private final OkHttpClient client;
-    private Logger logger;
+    private final Logger logger = LoggerFactory.getLogger(DefaultHttpClient.class);
```

Comment:
This class no longer even needs a Logger instance, from what I can see.

Reply:
Yes, I meant to remove that. Good catch